supply/lib/supply/uploader.rb

```diff
@@ -124,24 +124,22 @@ def fetch_track_and_release!(track, version_code, status = nil)
     end
 
     def update_rollout
-      track, release = fetch_track_and_release!(Supply.config[:track], Supply.config[:version_code], Supply::ReleaseStatus::IN_PROGRESS)
-      UI.user_error!("Unable to find the requested track - '#{Supply.config[:track]}'") unless track
-      UI.user_error!("Unable to find the requested release on track - '#{Supply.config[:track]}'") unless release
+      track_name = Supply.config[:track]
+      in_progress_status = Supply::ReleaseStatus::IN_PROGRESS
+      track, release = fetch_track_and_release!(track_name, Supply.config[:version_code], in_progress_status)
+      UI.user_error!("Unable to find the requested track '#{track_name}' with status in progress (#{in_progress_status})") unless track
+      UI.user_error!("Unable to find the requested release in track '#{track_name}' with status in progress (#{in_progress_status})") unless release
 
       version_code = release.version_codes.max
 
-      UI.message("Updating #{version_code}'s rollout to '#{Supply.config[:rollout]}' on track '#{Supply.config[:track]}'...")
+      UI.message("Updating #{version_code}'s rollout to '#{Supply.config[:rollout]}' in track '#{track_name}'...")
 
-      if track && release
```

Comment:
It's unnecessary to check for `track && release` here when we already run `UI.user_error!(...) unless ...` for both.